src/Pages/Profile/MemberView/GetApiKeyModal.js

```diff
@@ -0,0 +1,67 @@
+import React, {useState, useEffect} from 'react';
+import { getApiKey } from '../../../APIFunctions/User';
+
+const GetApiKeyModal = (props) => {
+  const { bannerCallback = () => {} } = props;
+  const [apiKey, setApiKey] = useState('');
+
+  const INPUT_CLASS_NAME = 'indent-2 block w-full rounded-md border-0 py-1.5 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:  focus:ring-2 focus:ring-insetfocus:ring-indigo-600 sm:text-sm sm:leading-6 text-white';
```

Comment:
why is this here, if only used once but for some reason all others dont have their css defined

Reply:
fixeddddd!!!!!!!!!!!!!!!!!!!!!!